docs/src/main/paradox/bootstrap/kubernetes-api.md

```diff
@@ -1,7 +1,5 @@
 # Kubernetes API
 
-An example project that can be deployed to Kubernetes via minikube is in `bootstrap-demo/kubernetes-api`.
```

Comment:
perhaps link back from this file and `kubernetes.md` to the kubernetes section of `recipes.md`?

Reply:
:+1: